utils/events/chat_helper_ticket_sys.py

```diff
@@ -1314,6 +1314,16 @@ async def sendCHTKTView(self, interaction: discord.Interaction):
             > {Emoji.ss_arrow} Send a picture of your assignment title in your direct messages as per the bot instructions.""",
             view=TicketButton(self.bot),
         )
+    
+    @app_commands.command(
+        name="title", description="Prompts for assignment title."
+    )
+    @app_commands.guilds(MainID.g_main)
```

Comment:
`@app_commands.guilds(MainID.g_main)`
- The guild check is fine since we only expect this command to be used in the main server, so no harm in using it.